test/capabilities.spec.js

```diff
@@ -47,7 +47,7 @@ describe('Capabilities', function() {
     await page.goto(server.EMPTY_PAGE);
     expect(await page.evaluate(() => window.testStatus)).toBe('SUCCESS');
   });
-  it.fail(WEBKIT)('should play video', async({page, server}) => {
+  it.fail(WEBKIT && !LINUX)('should play video', async({page, server}) => {
```

Comment:
wait, we don't play video on mac / win?

Reply:
According to Dmitry on mac there is a problem with our test server which doesn't support range query, so we should fix that. I will check if it works on windows and enable there separately.

Reply:
Reverting the test change for now, will enable it along with the roll.